manifests/application/migrator.pp

```diff
@@ -66,9 +70,13 @@
     $service_group = 'root',
     $migrator_version = 'latest',
     $jsvc_version = 'latest',
+    $migrator_home_dir = '/usr/share/migrator',
+    $migrator_config_dir = '/etc/migrator',
+    $migrator_log_dir = '/var/log/migrator',
+    $generate_log4j = false,
     $java_home = undef,
-    $jar_path = '/var/lib/jbossas/standalone/deployments/lightblue-migrator-consistency-checker-*.jar',
-    $service_log_file = 'console.log',
+    $jar_path = '/usr/share/jbossas/standalone/deployments/lightblue-migrator-consistency-checker-*.jar',
```

Comment:
That is true. It has always been this way, I am not sure of the original reasoning.